### ISSUE TITLE
File ownership not set correctly, specifically for .env file

### ISSUE THREAD

**Issue by FeedTheChunk** [Zinidia/Pelinstaller#5]
After using the script to install the Panel, I was unable to save setting changes in the Panel.  For instance Re-Capcha.

error message in part "SQLSTATE[HY000] [1045] Access denied "

Investigating the /var/www/pelican directory I noticed that not all files were owned by www-data www-data.  Sorry I didn't jot down exactly which ones but there were 3 or 4 at least in that directory. I did not check subdirectories. One of those files being the .env file which was owned by root.

Running  chown -R www-data:www-data /var/www/pelican set the ownership's correctly and the panel worked as expected.

ftc

**Comment by Zinidia** [comment]
Hi, I went ahead and implemented patch ffeeb84 for this. Pelican seems to create some files that may be owned by the incorrect linux user after the installation. If you notice similar issues, please run the following to find the files owned by the incorrect user and we may be able to add bypasses like I did for a3cd6d4
```
find /var/www/pelican -not -user www-data -exec ls -ld {} \;
```

**Comment by FeedTheChunk** [comment]
The Distro was  Ubuntu 22.04.  Like I said I didn't note down exactly all files that weren't owned by www-data.  I see you closed this which is fine.  I've already ran chown to correct the problems.  If I run into this again during my testing I'll post an update with more exact information.

c
